app/models/contact.rb

```diff
@@ -1,27 +1,27 @@
 class Contact < MailForm::Base
-  attribute :name,      :validate => true
-  attribute :email,     :validate => /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i
-  attribute :restroom_id, :allow_blank => true
-  attribute :restroom_name, :allow_blank => true
+  attribute :name, validate: true
+  attribute :email, validate: /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i # rubocop:disable Style/RedundantRegexpEscape, Style/RedundantRegexpCharacterClass
```

Comment:
I believe this regex could be replaced with `URI::MailTo::EMAIL_REGEXP`, but I'm not positive.

Reply:
I tried to look this up and ended up down a rabbit hole.

Apparently validating emails is a controversial subject. Some folks are saying you should just make sure there's at least one `@` sign in the form.

I might say "And there should be stuff before and after the `@` sign", and "there should be a dot after the `@` sign", and "three or more characters other than the dot should follow the `@` sign...

https://stackoverflow.com/questions/22993545/ruby-email-validation-with-regex

https://medium.com/hackernoon/the-100-correct-way-to-validate-email-addresses-7c4818f24643

When I do `puts URI::MailTo::EMAIL_REGEXP` in `irb`, I get the following:

```
(?-mix:\A[a-zA-Z0-9.!\#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*\z)
```

Hmmmmm...

Reply:
I imagine `URI::MailTo::EMAIL_REGEXP` is the better way to do this, but I would probably do a followup PR, as it's not related to linting and changes functionality of the code.